tests/puppeteer/campaigns/utils/files.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
   async doesFileExist(fileName, timeDelay = 5000, isPartialName = false, fileExtension = '') {
     let found = false;
     for (let i = 0; i <= timeDelay && !found; i += 100) {
-      await (new Promise(resolve => setTimeout(resolve, 10)));
+      await (new Promise(resolve => setTimeout(resolve, 100)));
```

Comment:
Why 100 instead of 10?
Maybe add a comment :thinking:

Reply:
Because it doesn't work in my local, the test run very quickly and does not wait for the file to be downloaded.